tests/test_logging/expected.txt

```diff
@@ -1,4 +1,10 @@
-           INFO     Read configuration files:                                                                                                                                                                                                       config.py:  
-           INFO                                                                                                                                                                                                                         scene_file_writer.py:   
-                    File ready at                                                                                                                                                                                                                               
+           INFO     Read configuration files: ['C:\\Users\\User\\OneDrive\\Bureau\\Programmation\\PYTHON\\MANIM-DEV\\manim\\manim\\default.cfg', 'C:\\Users\\User\\AppData\\Roaming\\Manim\\manim.cfg',                                            config.py:   
+                    'C:\\Users\\User\\OneDrive\\Bureau\\Programmation\\PYTHON\\MANIM-DEV\\manim\\tests\\tests_data\\manim.cfg']                                                                                                                                 
```

Comment:
Because of the way the tests are run, the test requires that no file paths should be in the expected log.

This is because the paths will differ from user to user and operating system to operating system.

I suggest you remove _all_ file paths completely.

Reply:
Yes, I'm dumb.
I removed that first, but idk what comes through my head but I forget to remove that again.

Thank you for the catch !